### PR TITLE
docs: :memo: Update documentation to point to the new CDN link

### PR DESCRIPTION
**Pull request by nabilridhwan**
Updated the documentation link from the old react-query to the new @tanstack/react-query

### CODE REVIEW

**Comment by nabilridhwan** [comment]
Fix #3852

File: docs/installation.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ React Query is compatible with React v16.8+ and works with ReactDOM and React Na
 If you're not using a module bundler or package manager we also have a global ("UMD") build hosted on the [unpkg.com](https://unpkg.com) CDN. Simply add the following `<script>` tag to the bottom of your HTML file:
 
 ```html
-<script src="https://unpkg.com/react-query/dist/react-query.production.min.js"></script>
+<script src="https://unpkg.com/@tanstack/react-query@4.0.0/build/umd/index.production.js"></script>
```
Comment (TkDodo) [inline docs/installation.md:27]: let's point at the latest v4:

```suggestion
<script src="https://unpkg.com/@tanstack/react-query@4/build/umd/index.production.js"></script>
```

**Comment by TkDodo** [comment]
PR needs to go from `main` to `main` please. Default branch in github is still wrong, sorry for that

**Comment by nabilridhwan** [comment]
Sorry!

**Comment by TkDodo** [comment]
it needs to go from `main` to `main` please. can you make a new PR?

**Comment by nabilridhwan** [comment]
Closing down this PR

**Pull request closed without merging**